### DISCUSSION TITLE
v2.2.1 - PreRelease

### DISCUSSION THREAD

**Discussion by Wenn0101** [sparkfun/Arduino_Apollo3#452] [General]
Changelist:
-Fix problems with A1-A5 on Redboard form factor
-Fix error in SPI which resulted in the object not being deleted
-Reduce time of critical section in pwm_output (analog/servo out)
-Allow for RingBuffer.h to be included directly in projects/libraries
-Added functions interrupts() and noInterrupts()
-Update mbed core to clear serial errors
-update Example10 to show chip ID

<hr /><em>This discussion was created from the release <a href='https://github.com/sparkfun/Arduino_Apollo3/releases/tag/v2.2.1-PreRelease'>v2.2.1 - PreRelease</a>.</em>

**Comment by nigelb** [comment]
Hi @Wenn0101 I am not sure when you bump the version numbers, but could you please bump `package.json` at the same time.
I can submit a pull request if needed.

**Reply by Wenn0101** [reply]
Yes will do
